.github/workflows/ets-from-source.yml

```diff
@@ -16,15 +16,11 @@ jobs:
   test-with-edm:
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest, windows-latest]
         toolkit: ['pyside6']
+        runtime: ['3.8']
```

Comment:
Added for parity with the `test-with-edm` workflow, and to ease adding 3.11 testing later on (when the necessary dependencies are present in EDS).